test/build.gradle

```diff
@@ -19,6 +19,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.3.31'
```

Comment:
Given the PR is already one year old, this should be updated (https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-stdlib)